pwa/src/components/logs/logTable/logTable.tsx

```diff
@@ -275,11 +261,12 @@ export const LogTable: React.FC<LogTableProps> = ({ id = null, query= null }) =>
                             return (<>
                               {log.requestContent
                                 ?
-                                <JsonCode body={log.requestContent} />
+                                <CodeBlock body={log.requestContent} />
                                 :
                                 <p className="utrecht-paragraph">No content found</p>}
                             </>)
-                          }
+                          },
+                          backgroundColor: 'black'
```

Comment:
This gives Typescript errors:

```
Type '{ title: string; id: string; render: () => Element; backgroundColor: string; }' is not assignable to type 'Record<"id" | "title" | "render", any>'.
  Object literal may only specify known properties, and 'backgroundColor' does not exist in type 'Record<"id" | "title" | "render", any>'.
```

Also, the background-color does not seem to be set in the accordion (it's still white)

Reply:
This works fine locally so I think u have to update your packages with 'npm install'